### ISSUE TITLE
Crash Vue Language Server if "strictPropertyInitialization": true

### ISSUE THREAD

**Issue by skycrazyk** [vuejs/vetur#1072]
## Info

- Platform: Linux Ubuntu 18.04.1 LTS
- Vetur version: 0.14.5
- VS Code version: 1.30.2

### Error message from Panel

```
Vetur initialized
/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:76414
                throw e;
                ^

Error: Debug Failure. Unhandled declaration kind! GetAccessor
    at getTypeOfVariableOrParameterOrProperty (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:29505:30)
    at getTypeOfSymbol (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:29678:24)
    at checkPropertyInitialization (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:46947:36)
    at checkClassLikeDeclaration (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:46776:17)
    at checkClassDeclaration (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:46678:13)
    at checkSourceElement (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:47821:28)
    at Object.forEach (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:1762:30)
    at checkSourceFileWorker (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:47958:20)
    at checkSourceFile (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:47937:13)
    at getDiagnosticsWorker (/home/crazyk/.vscode/extensions/octref.vetur-0.14.5/server/node_modules/typescript/lib/typescript.js:48002:17)
[Error - 6:33:44 PM] Connection to server got closed. Server will not be restarted.
```
### Screenshot

![output screen](https://i.imgur.com/nKyExJ7.png)

## Reproducible Case

I just start VSCode and open any `.vue` file. I try fix this error. The error does not occur if in the `tsconfig.json`, set `"strictPropertyInitialization": false` (or just remove it).

**Comment by octref** [comment]
This is only available in TS 2.8+. Will be fixed with #682.

**Comment by octref** [comment]
See https://github.com/vuejs/vetur/issues/682#issuecomment-474971395
